lib/Dancer2/Core/App.pm

```diff
@@ -7,6 +7,7 @@ use Scalar::Util       'blessed';
 use Module::Runtime    'is_module_name';
 use Return::MultiLevel ();
 use Safe::Isa;
+use Scalar::Lazy       ();
```

Comment:
Oops - that's a leftover for another experiment that I missed removing :( Thanks for spotting it!